app/services/subcases-service.js

```diff
@@ -54,6 +54,10 @@ export default class SubcasesService extends Service {
             date: geplandeStart,
           });
           if (phaseData.decisionResultId) {
+            // Why don't we load in this data in the custom-subcases-service?
```

Comment:
Maybe this shouldn't actually be a comment in the code, but just wrote it down somewhere when it occurred to me :sweat_smile:

Reply:
Good question... The performance already seems quite OK as it is though. So let's leave this in there for now.

Reply:
Instead of a question in comments, you could create a Jira ticket and mention it here
`// KAS-xxxx load in this data in the custom-subcases-service instead`

Part of the audit last year also suggested to not have "TO DO" or question comments in code unless there is linked ticket available so we should try to avoid adding them.